config/vufind/Blender.ini

```diff
@@ -0,0 +1,260 @@
+; equal sign is applied.  If the box is not checked, the filter is not applied.
+; The value on the right side of the equal sign is the text to display to the
+; user.  It will be run through the translation code, so be sure to update the
+; language files appropriately.
+;
+; Leave the section empty if you do not need checkbox facets.
+[CheckboxFacets]
+; blender_backend facet is special and allows limiting of results to one of the
+; actual backends:
+;blender_backend:Solr = "Items in Library"
+;blender_backend:SolrAuth = "Authors"
+;blender_backend:Primo = "Electronic Material (Ex Libris CDI)"
+;blender_backend:EDS = "Electronic Material (EBSCO EDS)"
+peer_reviewed:1 = peer_reviewed
+fulltext:1 = online_resources
+
+; These settings affect the way the [Results] facets are displayed
+; If using facets at the top of search results you have more room for text.
+[Results_Settings]
+; By default, how many values should we show for each facet? (-1 for no limit)
+facet_limit        = 30
+; Override facet_limit on a per-field basis using this array:
+;facet_limit_by_field[format] = 50
+
+; By default, the side facets will only show 6 facets and then the "show more"
+; button. This can get configured with the showMore settings.
+; You can use the * to set a new default setting.
+showMore[*] = 6
+; Or you can set a facet specific value by using the facet name as index.
+;showMore['format'] = 10
+
+; Show more facets in a lightbox (paginated, no limit)
+; If false, facets expand in side bar to show facets up to the above limit
+; If "more", facets expand and offer an option at the bottom to open the lightbox
+; If true, facets immediately open in the lightbox
+showMoreInLightbox[*] = more
+lightboxLimit = 100 ; page size for the lightbox
+
+; Number of rows of values for top facets to show above the "more" button
+top_rows = 2
+; Should we show "exclude" links for some or all of the facets? Set to * for
+; all facets, use a comma-separated list to show for some of the facets, set
+; to false or omit to disable "exclude" links.
+exclude = blender_backend
+; Should we OR together facets rather than ANDing them? Set to * for
+; all facets, use a comma-separated list to apply to some of the facets, set
+; to false or omit to disable ORed facets.
+orFacets = *
+; Do we want any facets to be collapsed by default?
+collapsedFacets = *
+
+; This can be used to sort specific facet fields alphabetically by index value
+; (which normally results in alphabetical order).
+; Please note: This sorts natively in the Solr index using untranslated values,
+; so if you are using facet translation, your values may not always display in
+; the expected order.
+;sorted_by_index[] = publish_date
+
+; Most of these settings affect the way the [Advanced] facets are displayed; the
+; translated_facets setting affects facets globally.
+[Advanced_Settings]
+facet_limit      = -1      ; how many values should we show for each facet?
```

Comment:
I don't think we need this setting, since we don't have an advanced search screen for this to apply to.